### PR TITLE
Fixes #2352 Blue point does not move with me, strange grey point does

### LINKED ISSUE #2352: Blue point does not move with me
screenshot_20190125-192150
Expected: The blue point is supposed to follow me.
Actually: The blue point stays where it appeared when I launched the Nearby activity. The grey point follows me.

### PR DESCRIPTION
**Pull request by shabrikaruna**
Fixes #2352 Blue point does not move with me, strange grey point does

What changes did you make and why?

The grey circle is removed as there is no use of that and the blue location marker is working fine with location updating.

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java [hunk 1]
```diff
@@ -131,13 +128,24 @@
...
+    JsonKvStore directKvStore;
+    @Inject
+    @Named("default_preferences")
+    BasicKvStore defaultKvStore;
+    @Inject
+    BookmarkLocationsDao bookmarkLocationDao;
+    @Inject
+    ContributionController controller;
+    @Inject
+    Gson gson;
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java]: Was this change really necessary?

Comment (shabrikaruna) [reply app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java]: Changes was not done by me when i pulled the repository the changes has been imported

**Comment by nicolas-raoul** [comment]
Any progress on this? :-)
